tests/testthat/test-convert-gene-ids.R

```diff
@@ -1,30 +1,86 @@
-# read in SCE for testing
-sce <- readRDS(test_path("data", "scpca_sce.rds"))
```

Comment:
I wonder whether it's starting to get worthwhile to have a [`setup.R`](https://testthat.r-lib.org/articles/special-files.html#setup-files) script that at least load the sce file? Separate PR, if at all, though.